db/seeds.rb

```diff
@@ -5,3 +5,49 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+rentals = []
```

Comment:
I don't think you use the `rentals` variable after creating the 3 rentals - I think you can just create them without assigning to a variable, no?